package.json

```diff
@@ -43,19 +43,23 @@
     "yargs": "^16.2.0"
   },
   "devDependencies": {
+    "@eslint/js": "^9.11.1",
     "all-contributors-cli": "^4.11.2",
     "aud": "^2.0.4",
     "babel-cli": "^6.26.0",
     "babel-core": "^6.26.3",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "codecov": "^2.3.1",
     "commitizen": "^2.10.1",
     "create-require": "^1.1.1",
     "cz-conventional-changelog": "^2.1.0",
-    "eslint": "^7 || ^8.2.0",
-    "eslint-plugin-json": "^3.1.0",
+    "eslint": "^8.2.0 || ^9.0.0",
+    "eslint-plugin-json": "^4.0.1",
     "ghooks": "^2.0.4",
+    "globals": "^15.9.0",
     "in-publish": "^2.0.1",
+    "lodash": "^4.17.21",
```

Comment:
lodash is a CVE magnet; if we can't use `Object.assign` or the `extend` package, let's use the more granular `lodash.merge`.

Reply:
i understand, i changed